src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java

```diff
@@ -313,7 +315,7 @@ public final void setAlpha(float opacity)
 	public boolean dispatchTouchEvent(MotionEvent e)
 	{
 		String originalIndent = _indent;
-		Log.i("", _indent + "    + " + this.toString());
+		Log.i(LOGTAG, _indent + "    + " + this.toString());
```

Comment:
Do we need to keep this logging in production code?

Reply:
+1 
(Hint: we do not.)

Reply:
This code is here to ease debug as debug Android is not really easy, and the whole mothod is commented-out. I would keep it for future debug, however the `LOGTAG` should be put here, as this method is trying here to dump the indent the log to follow the visual tree.

Reply:
> and the whole method is commented-out.

?

(PS: Debugging Java code is actually really easy :p )

Reply:
Those `Log` lines were already there. I just added a tag, so they can be easily filtered/searched.
And, this entire method is commented out with `/**/` outside of this snippet.